src/Compilers/Core/CodeAnalysisTest/DefaultAnalyzerAssemblyLoaderTests.cs

```diff
@@ -682,6 +689,7 @@ public void AssemblyLoading_MultipleVersions(bool shadowLoad)
                 Assembly epsilon = loader.LoadFromPath(testFixture.Epsilon.Path);
                 var e = epsilon.CreateInstance("Epsilon.E")!;
                 e.GetType().GetMethod("Write")!.Invoke(e, new object[] { sb, "Test E" });
+                var actual = sb.ToString();
```

Comment:
Did moving this statement cause behavior to change?

Reply:
No. This method used to be a mix of `#if NETCOREAPP` and `if (ExecutionCondition.IsCoreClr)` checks which seemed a bit odd. Just moved to `#if NETCOREAPP`. As part of that I needed to move that declaration up a bit higher so it was available in both configs.